src/Core/Prefab/GlobeView.js

```diff
@@ -163,6 +163,18 @@ export function createGlobeLayer(id, options) {
         return false;
     }
 
+    wgs84TileLayer.sumOccupancyColorLayers = (layerColors, layer) => {
+        layerColors.push(layer);
```

Comment:
Better avoid modifying arguments.
Change the signature of the function to `colorLayers` and let the caller passes the correct layer array.